src/noise-filter.js

```diff
@@ -110,62 +108,86 @@ export default class NoiseFilter {
         folder
             .add(this.mods.baseRoughness, 'mapTo', config.features)
             .name('baseRoughness')
-            .onChange((val) => (this.mods.baseRoughness['mapTo'] = val))
+            .onChange((val) => {
+                this.mods.baseRoughness.mapTo = val
```

Comment:
these aren't the slickest. I was thinking of using a factory function like:
```javascript
const createOnChange = (mod, field) => (val) => {
    this.mods[mod][field] = val
}
```
to be used like 
```javascript
.onChange(createOnChange('baseRoughness', 'mapTo')
```
but im worried it might become hard to debug if something went wrong, idk

Reply:
I think we tried that before and for some reason the factory function wouldn't work.  It's possible we were doing it wrong and there is a work around but at the time we concluded that hard coding was the way to go.